### DISCUSSION TITLE
Update CIPP

### DISCUSSION THREAD

**Discussion by Hannes7c** [KelvinTegelaar/CIPP#891] [Q&A] [answered]
Hello everyone,

I see there is a new version of CIPP available. What is the best way to perform this update?

Thank you

**Comment by KelvinTegelaar** [comment, marked as answer]
Have you tried checking the docs at https://cipp.app

**Comment by henrivisserslannet** [comment]
I followed the manual. It works for the Frontend Version. But the Backend API version says: Latest 4.7.4. and current 3.5.0. 

The Sync Fork part for the API is done. But the API still doesnt update. Any tips where to start with troubleshooting would be fine.

**Reply by KelvinTegelaar** [reply]
Support is only for sponsors; if you are not sponsoring you'll have to use the documentation to resolve your issues.

**Reply by equeck21** [reply]
I had to go in the function app in azure. Then Deployment center on the lefthand menu, then click the sync button. After that it finally updated that backend api version.
